samples/shields/x_nucleo_iks4a1/sensorhub1/src/main.c

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright (c) 2019 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/sensor.h>
+#include <stdio.h>
+#include <zephyr/sys/util.h>
+
+#ifdef CONFIG_LSM6DSV16X_TRIGGER
+static int lsm6dsv16x_acc_trig_cnt;
+
+static void lsm6dsv16x_acc_trig_handler(const struct device *dev,
+				     const struct sensor_trigger *trig)
+{
+	sensor_sample_fetch_chan(dev, SENSOR_CHAN_ALL);
+	lsm6dsv16x_acc_trig_cnt++;
+}
+#endif
+
+static void lsm6dsv16x_config(const struct device *lsm6dsv16x)
+{
+	struct sensor_value odr_attr, fs_attr;
+
+	/* set LSM6DSV16X accel sampling frequency to 208 Hz */
+	odr_attr.val1 = 208;
+	odr_attr.val2 = 0;
+
+	if (sensor_attr_set(lsm6dsv16x, SENSOR_CHAN_ACCEL_XYZ,
+			    SENSOR_ATTR_SAMPLING_FREQUENCY, &odr_attr) < 0) {
+		printk("Cannot set sampling frequency for LSM6DSV16X accel\n");
+		return;
+	}
+
+	sensor_g_to_ms2(16, &fs_attr);
+
+	if (sensor_attr_set(lsm6dsv16x, SENSOR_CHAN_ACCEL_XYZ,
+			    SENSOR_ATTR_FULL_SCALE, &fs_attr) < 0) {
+		printk("Cannot set fs for LSM6DSV16X accel\n");
```

Comment:
```suggestion
		printk("Cannot set full scale for LSM6DSV16X accel\n");
```

(you spell it fully for the other one)

Reply:
ok, done everything you highligthed. Thanks for your patience in reviewing it!